azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/callingcompositedemoapp/robots/CallScreenRobot.kt

```diff
@@ -64,11 +64,11 @@ class CallScreenRobot : ScreenRobot<CallScreenRobot>() {
     fun clickLeaveCall() {
         val idlingResource = ViewIsDisplayedResource()
         idlingResource.waitUntilViewIsDisplayed {
-            UiTestUtils.checkViewIdIsDisplayed(R.id.azure_communication_ui_call_leave_confirm)
+            UiTestUtils.checkViewWithTextIsDisplayed("Leave call?")
```

Comment:
Do we want to use hardcoded string rather than string from string resource?

Reply:
@Mohtasim  it is intended to be a guard against any scenario where the string changes, eg. it may be swapped with another R.string in the future